### PR TITLE
Edit documentation for user

### PR DESCRIPTION
**Pull request by felixhalim**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,41 +2,200 @@
...
+
+* [Quick Start](#quick-start)
+* [Features](#features)
+    * [Create new trip: `new trip`](#creating-new-trip-new-trip)
+    * [View help: `help`](#viewing-help-help)
+    * [Add a place: `add`](#adding-a-place-add)
+    * [List all places: `list /p`](#listing-all-places-list-p)
+    * [Search for place: `search`](#searching-for-place-search)
+    * [Delete place: `delete /p`](#deleting-place-delete-p)
+    * [Edit budget: `budget`](#editing-budget-budget)
```
Comment (ShawnTanzc) [inline docs/UserGuide.md:15]: General notes: 
+ 'convert /to'
+ 'edit functions' (ForEx Rate, ForEx Abbreviation)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -2,41 +2,200 @@
...
+3. Copy the file to a folder where you want to run it from.
+4. Using the terminal, navigate to the path of `trippie.jar`.
+5. Run `java -jar trippie.jar`
+6. If the setup is correct, you should see a welcome message. Then, it is now ready to go.
 
 ## Features 
 
-{Give detailed description of each feature}
+### Creating new trip: `new trip`
+Creates a new trip. You will be prompted to enter some details (i.e. name, start date, exchange rate, and budget) on
```
Comment (ShawnTanzc) [inline docs/UserGuide.md:35]: include foreign currency abbreviation

**Review by ShawnTanzc — APPROVED** [review]
LGTM, have to add new functions once merge

**Review by ivanderjmw — APPROVED** [review]
Looks like a good user guide. Pay attention to typos.

**Pull request merged**